apps/web/src/pages/Skills/UpdateSkillPage.tsx

```diff
@@ -203,7 +203,7 @@ export const UpdateSkillForm = ({
   const skillFamilyOptions: Option<string>[] = sortedFamilies.map(
     ({ id, name }) => ({
       value: id,
-      label: name?.[locale] || "",
+      label: name?.[locale] ?? "",
```

Comment:
Pre-existing, I wonder if this should be a "not found" string instead?

Reply:
Updated in [fix localized names null fallbacks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/37d38ad80f04bf3718d0ba3b6d072894c2fe03c1)